q_yaar_platform/q_yaar/forms.py

```diff
@@ -0,0 +1,27 @@
+import uuid
+
+from django.contrib.auth import get_user_model, authenticate
+from django.contrib.auth.forms import AuthenticationForm
+
+UserModel = get_user_model()
+
+
+class CustomAdminAuthenticationForm(AuthenticationForm):
```

Comment:
Custom admin login form. Pretty much the same, just typecasts the user name field to uuid.